SwarmHeader header

int8 fcuMode
int8 fcuStatus

float64 timeSinceLaunch
float64 distanceFromLaunch
int16   numGpsSatelites

bool pixhawkReady
bool arduinoReady
bool hasGPSFix
bool hasWiFi